content/en/docs/Univariate/MCMCglmm/_index.md

```diff
@@ -217,3 +218,6 @@ HPDinterval(model1.2$VCV[, "id"])
 
 
 
+```
+## [1] TRUE TRUE TRUE TRUE
```

Comment:
Not sure why those messages/outputs are printed? Do we want them?

Reply:
due to problem with plots not found
resolved with plot at root